hudi-spark-datasource/hudi-spark2/pom.xml

```diff
@@ -29,6 +29,7 @@
 
   <properties>
     <main.basedir>${project.parent.parent.basedir}</main.basedir>
+    <scala.version>${scala11.version}</scala.version>
```

Comment:
why is this necessary? spark2 can be used with 2_12 right?

Reply:
Yes, I will remove this.